src/main/java/org/fundacionjala/coding/manuel/movies/Customer.java

```diff
@@ -0,0 +1,88 @@
+package org.fundacionjala.coding.manuel.movies;
+
+
+import java.util.Enumeration;
+import java.util.Vector;
+
+/**
+ * Created by Juan Manuel on 3/18/2018.
+ */
+public class Customer {
+    private String name;
+    private Vector rentals;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace the synchronized class "Vector" by an unsynchronized one such as "ArrayList" or "LinkedList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1149)